### PR TITLE
add structured data.

### LINKED ISSUE #181: Add schema.org data to the Careers page
See [this code](https://github.com/gitpod-io/website/blob/master/src/components/careers/ExpandableJob.tsx#L125) from the previous website to get started.

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #181

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/758"></a>

### CODE REVIEW

**Comment by nisarhassan12** [comment]
I have tested this and this is what the results look like see https://search.google.com/test/rich-results?id=hcTkw8vRxHD-0TZpAlmeow:

image

File: src/components/careers/structured-data.svelte [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+  import type { Career } from "../../types/career.type";
+  export let career: Career;
+  const { title, intro, paragraphs, lists, textAfterTheLists, date } = career;
+
+  const applicationEmail = `mailto:career@gitpod.io?subject=Application as ${title}`;
```
Comment (mikenikles) [inline src/components/careers/structured-data.svelte:6]: This is the 4th place where we hard-code the email address. It'd be fantastic if we could refactor the code as part of this PR to define the email address in single place and use it where it is currently hard-coded.

Comment (nisarhassan12) [reply src/components/careers/structured-data.svelte]: Thanks. Done.

File: src/utils/constants.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const CAREER_EMAIL = 'career@gitpod.io'
```
Comment (mikenikles) [inline src/utils/constants.ts:1]: I'm surprised Prettier didn't add a semicolon when you committed your changes.

Could you show me a screenshot of the terminal output the next time you commit changes? I'd like to understand why Prettier didn't do its job.

Comment (nisarhassan12) [reply src/utils/constants.ts:1]: I usually use the Source control panel and I suspect the pre-commit hook does not run this way for some reason. 
Screen Shot 2021-07-27 at 3 33 50 PM

For testing I just used termnial to make a commit and I can verify that the `pre-commit` hook ran 
Screen Shot 2021-07-27 at 3 36 34 PM

**Pull request merged**